doc/api_assets/style.css

```diff
@@ -93,7 +93,10 @@ em code {
   color: white !important;
   margin: 0 0 1em 0;
   font-family: "Lato", "Lucida Grande", "Lucida Sans Unicode", "Lucida Sans", Verdana, Tahoma, sans-serif;
-  font-weight: 700;
+  white-space: pre-wrap;
+  word-wrap: break-word;
+  padding: 1em;
+  line-height: 1.5;
```

Comment:
Ahh, you're right. Since it's not `pre` anymore, the wrapping is no longer an issue. The `padding` and `line-height` are still necessary though.